doc/user/oss_backup.md

```diff
@@ -0,0 +1,125 @@
+# Backups using Alibaba Cloud Object Storage Service (OSS)
+
+Etcd backup operator backs up the data of an etcd cluster running on Kubernetes to a remote storage such as Alibaba Cloud Object Storage Service (OSS). If it is not deployed yet, please follow the [instructions](walkthrough/backup-operator.md#deploy-etcd-backup-operator) to deploy it, e.g. by running
+
+```sh
+kubectl apply -f example/etcd-backup-operator/deployment.yaml
+```
+
+## Setup Alibaba Cloud backup account, OSS bucket, and Secret
+
+1. Login [Alibaba Cloud Console](https://www.alibabacloud.com) (or [Aliyun Console](https://www.aliyun.com/) if you are in China) and create your own [AccessKey](https://www.alibabacloud.com/help/doc-detail/29009.htm) which gives you the AccessKeyID (AKID) and AccessKeySecret (AKS). You can optionally create an Object Storage Service ([OSS](https://www.alibabacloud.com/help/doc-detail/31947.htm)) bucket for backups.
+2. Create a secret storing OSS AK/SK in kubernetes.  
```

Comment:
Update the acronyms. Also the Access credentials are not specific/related to the OSS right? That's why i rephrased it to not say OSS.
```suggestion
2. Create a secret storing your AKID and AKS in Kubernetes.  
```